package.json

```diff
@@ -6,19 +6,41 @@
   "license": "MIT",
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "lerna run libbuild --stream",
-    "build-storybook": "lerna run libbuild --stream --scope=@cmpsr/components && lerna run build-storybook --stream",
-    "test": "lerna run test --stream",
-    "test:watch": "lerna run --parallel test:watch --stream",
-    "test:coverage": "lerna run --parallel test:coverage --stream",
+    "build": "turbo run libbuild",
```

Comment:
can we parallelise this?

Reply:
I dont think this is the best solution since it skips dependencies. Turbo tries to parallel the tasks as much as possible. https://turborepo.org/docs/reference/command-line-reference#--parallel